libs/net/dal/Services/UserService.cs

```diff
@@ -134,6 +134,8 @@ public IEnumerable<User> GetUserUpdateHistory()
     public User? FindByUserKey(string key)
     {
         return this.Context.Users
+            .Include(u => u.MediaTypes)
```

Comment:
When impersonating an indirect user, it will now correctly apply the blocked content.